wpk/run.sh

```diff
@@ -133,7 +133,7 @@ main() {
 
     NO_COMPILE=false
     # Get Wazuh
-    git clone ${REPOSITORY} ${DIRECTORY} -b ${REFERENCE} --depth=1 || exit 1
+    curl -sL https://github.com/wazuh/wazuh/tarball/${REFERENCE} | tar zx
```

Comment:
You can remove the `.git` from the URL and use that variable with `curl`:
```suggestion
    curl -sL ${REPOSITORY}/tarball/${REFERENCE} | tar zx
```

Reply:
Done in aade64a